### PR TITLE
migrate

### LINKED ISSUE #20: Migrate to self-hosted action runner
## Prevent hitting monthly limit

... by using our own server for action processing.

## Description of a possible solution

Change `ubuntu-latest` to `self-hosted` `ubuntu` `Linux` or `16.04`

## Alternatives considered

Change runner keyword to `latest`, by that is misinforming since we run 16.04

### PR DESCRIPTION
**Pull request by Toeffe3**
## Purpose: `Self-hosted`

Read #20 

## Approach

Find and replace

### CODE REVIEW

File: .github/workflows/Issue_checklist_checker.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on: issues
 jobs:
   checkIssue:
     name: Check Issue for Incomplete Checklist
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
```
Comment (Andreasgdp) [inline .github/workflows/Issue_checklist_checker.yml:8]: Why does this say self-hosted, when all of the others run on "vds2019x11"?

Comment (Toeffe3) [reply .github/workflows/Issue_checklist_checker.yml:8]: Because VSC file search (& replace) sucks

File: .github/workflows/issue_labeler.yml [hunk 2]
```diff
@@ -5,68 +5,8 @@ on:
...
-        with:
-          keywords: '["?", "question", "Question"]'
-          labels: '["question"]'
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-      - uses: Naturalclar/issue-action@v1.0.0
-        with:
-          keywords: '["CSS", "css", "Css", "styling"]'
-          labels: '["CSS"]'
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-      - uses: Naturalclar/issue-action@v1.0.0
```
Comment (Andreasgdp) [inline .github/workflows/issue_labeler.yml:65]: Why has this been removed? I get why you have gone over and cleaned up the categorizing, but this I feel like does not need to be removed.

**Review by Andreasgdp — CHANGES_REQUESTED** [review]

File: .github/workflows/issue_labeler.yml [hunk 3]
```diff
@@ -5,68 +5,8 @@ on:
...
-        with:
-          keywords: '["- [ ] #", "- [x] #"]'
```
Comment (Andreasgdp) [inline .github/workflows/issue_labeler.yml:67]: Why has this been removed? I get why you have gone over and cleaned up the categorizing, but this I feel like does not need to be removed.

**Review by Andreasgdp — CHANGES_REQUESTED** [review]

File: .github/workflows/issue_labeler.yml (hunk 3 above)
Comment (Toeffe3) [reply .github/workflows/issue_labeler.yml:67]: I don't know why, I think I update from master and it had conflict, I just kept master

Comment (Toeffe3) [reply .github/workflows/issue_labeler.yml:67]: i can see that it is github garbage -  all rules at still there

**Review by Toeffe3 — COMMENTED** [review]
I have compared this and the files in api and other are no differences that should have an effect

File: .github/workflows/issue_labeler.yml [hunk 4]
```diff
@@ -5,68 +5,8 @@ on:
...
-          labels: '["Grouped Task"]'
```
Comment (Andreasgdp) [inline .github/workflows/issue_labeler.yml:68]: This part right here has been removed, and I don't think it should.

Couldn't get github to select the right lines before in my comment.

**Review by Andreasgdp — CHANGES_REQUESTED** [review]

**Review by Andreasgdp — APPROVED** [review]
They work for now. Some changes will have to be made later on in order to speed up the actions.

**Pull request merged**